src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Windows/UnoQuickStart.Windows.csproj

```diff
@@ -20,32 +20,27 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.2.221109.1" />
 		<PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
-		<PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
-		<PackageReference Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
-		<PackageReference Include="Uno.WinUI" Version="4.7.0-dev.417" />
```

Comment:
We need to keep this one, as we still have users referencing `VisibleBoundsPadding` and other controls provided by the `Uno.UI.Toolkit` assembly.

Reply:
I removed all those that come transitively to avoid maintaining versions in two projects (both for us and for template consumers), but I can add back.

Reply:
Right. That makes sense, at least it's not coming from the uno package itself, that should do.